lib/rules/block-indentation.js

```diff
@@ -161,17 +175,33 @@ module.exports = class BlockIndentation extends Rule {
 
   visitor() {
     this._elementStack = [];
+    this.seen = new Set();
+    this.nodeToSource = new WeakMap();
 
     return {
+      Template(node) {
+        if (this.mode === 'fix') {
+          if (
+            node.body[0] &&
```

Comment:
Better way to write this is: `node.body.length > 0`

Reply:
Like I said thé indentation of block start IS validated only for thé first node while It should validated all. This code disappear if WE want to really fix this issue